### DISCUSSION TITLE
Is STM32H743VIT6 development bord (DevBox) supported ?

### DISCUSSION THREAD

**Discussion by ak-ferhat** [stm32duino/Arduino_Core_STM32#1567] [Variants] [answered]
Hi,
I am new in STM32 universe, and I have some knowledge about ESP32 and Arduino.
I plan to work with STM32H743VIT6 development board (DevBox) with can be seen here  [https://stm32-base.org/boards/STM32H743VIT6-STM32H7XX-M.html](https://stm32-base.org/boards/STM32H743VIT6-STM32H7XX-M.html)
I want to know if this card is supported.

Best regards,

**Comment by fpistm** [comment, marked as answer]
Hi @ak-ferhat 
currently no dedicated variants have been made for this board anyway you can try the generic H743VI variants.

**Reply by fpistm** [reply]
As the WeAct boards were added, DevEBox were very similar so I've added them.
https://github.com/stm32duino/Arduino_Core_STM32/pull/1573

**Reply by ak-ferhat** [reply]
Thank you
I'll give it a try.
